js/UrbanAirship.js

```diff
@@ -494,6 +494,32 @@ class UrbanAirship {
     return UrbanAirshipModule.isQuietTimeEnabled();
   }
 
+  /**
+   * Sets Autobadging on iOS, either to true or false. Badging is not supported for Android.
```

Comment:
I'd recommend rephrasing the first bit as "Enables or disables autobadging on iOS."

Reply:
Okie doke!